MAUI/Maps/vector-layers/polyline-layer.md

```diff
@@ -255,7 +255,7 @@ this.Content = maps;
 
 ## Stroke cap
 
-You can apply different stroke cap to each `MapPolyline` in the `Polylines` collection using the individual `MapPolyline.StrokeLineCap` property. The default value of the `MapPolylineLayer.StrokeLineCap` property is `StrokeLineCap.Butt`. The available values are `Butt`, `Round`, and `Square`.
+You can apply different stroke cap to each [`MapPolyline`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolyline.html) in the [`Polylines`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolylineLayer.html#Syncfusion_Maui_Maps_MapPolylineLayer_Polylines) collection using the individual [`MapPolyline.StrokeLineCap`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolyline.html#Syncfusion_Maui_Maps_MapPolyline_StrokeLineCap) property. The default value of the [`MapPolylineLayer.StrokeLineCap`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolyline.html#Syncfusion_Maui_Maps_MapPolyline_StrokeLineCap) property is `StrokeLineCap.Butt`. The available values are `Butt`, `Round`, and `Square`.
```

Comment:
API reference not included for `StrokeLineCap.Butt`, `Butt`, `Round`, and `Square`.

Reply:
It's a framework class. So not add any API reference.